en/components/for_of.md

```diff
@@ -0,0 +1,149 @@
+---
+title: igxFor
+_description: The Ignite UI for Angular product is providing virtual directive igxFor similar to ngFor, but it is creating scrollbars and only renders small chunks of the data.
```

Comment:
Ignite UI for Angular now exposes a virtual igxFor directive similar to ngFor, which virtualizes DOM object rendering by visualizing only the visible chunks of the data in the DOM.